src/translations/en.json

```diff
@@ -3229,6 +3229,7 @@
           "payload": "Payload (template allowed)",
           "publish": "Publish",
           "description_listen": "Listen to a topic",
+          "json_formatting": "Format JSON content",
```

Comment:
Some how this localization string does not show up in the UI, I have no idea what I have missed.